Tests/Suites/Extensions/Dictionary+TogglesTests.swift

```diff
@@ -11,14 +11,6 @@ final class Dictionary_TogglesTests: XCTestCase {
         XCTAssertTrue( (try emptyDictionary.convertToTogglesDataSource()).isEmpty)
     }
     
-    func testInvalidFormatDictionaryConversion() {
-        let invalidDictionary: [String: Any] = ["key": [4,5]]
-        
-        XCTAssertThrowsError(try invalidDictionary.convertToTogglesDataSource()) { error in
-            XCTAssertEqual(error as? Dictionary.ToggleConversionError, Dictionary.ToggleConversionError.unsupportedValueType("Array<Int>"))
-        }
-    }
-    
     func testInvalidFormatDictionaryConversion_multipleValues() {
```

Comment:
Given you've removed the method above, the suffix `_multipleValues` could go too.
It should also be renamed from `InvalidFormatConversion` to `UnsupportedValueType`.
Also, for consistency, the tests in this repo have a `test_` prefix.